test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/CreateChatCompletionResponseChoice.cs

```diff
@@ -39,6 +39,6 @@ internal CreateChoice(long index, ChatCompletionResponseMessage message, CreateC
         /// specified in the request was reached, `content_filter` if the content was omitted due to
         /// a flag from our content filters, or `function_call` if the model called a function.
         /// </summary>
-        public CreateChoiceFinishReason FinishReason { get; }
+        public CreateChatCompletionResponseChoiceFinishReason FinishReason { get; }
```

Comment:
This is union under anonymous model above: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L275